docs/apache-airflow/core-concepts/dags.rst

```diff
@@ -165,6 +165,14 @@ While both DAG constructors get called when the file is accessed, only ``dag_1``
 
 You can also provide an ``.airflowignore`` file inside your ``DAG_FOLDER``, or any of its subfolders, which describes patterns of files for the loader to ignore. It covers the directory it's in plus all subfolders underneath it. See  :ref:`.airflowignore <concepts:airflowignore>` below for details of the file syntax.
 
+In the case where the ``.airflowignore`` does not meet your needs and you want a more flexible way to control if a python file needs to be parsed by Airflow. You can plug your callable by setting ``might_contain_dag_callable`` in the config file.
```

Comment:
CAn you please clarify that this callable replaces the above "airflow" and "dag" check? It's not obvious and the reader might think that this check is "extra" on top of "airflow|dag".

Reply:
great call! thanks